recipes/categorical_anomaly.py

```diff
@@ -5,18 +5,23 @@
 import geopandas as gpd
 import tqdm
 
+import rasterio
+import rasterio.mask
+from rasterio import open
```

Comment:
since `open` is a pretty generic function (also conincides with a Python built-in one), why not always use it as `rasterio.open` (as you do below) so that it doesn't accidentally get used in place of intended built-in `open`

Reply:
Ah ya good point, I think I'd maybe gotten a little frustrated when some rasterio function wasn't working with just a import rasterio so I might've gone a little over board on the importing, and didn't clean them up perfectly. Will change!

Reply:
if you configure editor with linting (e.g. vs code + python plugin) it should point out unused imports for you